docs-src/docs/index.md

```diff
@@ -1,6 +1,12 @@
-![Layer0 Logo](/static/logo_rs.png)
+<!-- ![Layer0 Logo](/static/logo_rs.png) -->
+<p align="center">
```

Comment:
The point of markdown is to remove html

Reply:
I know. You can't center with markdown. It looks better imo.

Reply:
In the past we had a guy writing our docs and literally copy+pasted thousands of lines of html that we later had to remove by hand. Seeing anyone trying to add it back triggers some flashbacks haha. Given this is only a couple if lines, it's no big deal, but please do not make a habit of it!

Reply:
Haha got it. I agree with you and I'm a huge fan of markdown. I tried to do it in markdown and it's not possibly. :/